exporter/ststopologyexporter/internal/types.go

```diff
@@ -0,0 +1,41 @@
+package internal
+
+type Instance struct {
+	Type string `json:"type"`
+	URL  string `json:"url"`
+}
+
+type ComponentType struct {
+	Name string `json:"name"`
+}
+
+type ComponentData struct {
+	Name        string            `json:"name"`
+	Version     string            `json:"version"`
+	Layer       string            `json:"layer"`
+	Domain      string            `json:"domain"`
+	Environment string            `json:"environment"`
+	Labels      []string          `json:"labels"`
```

Comment:
I would drop labels from here. That's a legacy thing in our API that we only have because of the way the original agent reported data.